base/src/main/java/com/smartdevicelink/proxy/rpc/Choice.java

```diff
@@ -153,15 +154,18 @@ public void format(Version rpcVersion, boolean formatParams){
 
         if (rpcVersion == null || rpcVersion.getMajor() < 5){
 
-            // make sure there is at least one vr param
-            List<String> existingVrCommands = getVrCommands();
+            // this is added to allow the choice set manager to disable this functionality
+            if (!getIgnoreAddingVRItems()) {

```

Comment:
This can be a direct access to the variable as it's private and in the class. Unless you moved this to a static instance for all choice instances and then synchronized the read/write operations.